### ISSUE TITLE
JSON Schema

### ISSUE THREAD

**Issue by edbek** [sverweij/state-machine-cat#218]
Do you have a JSON Schema file to work with JSON files that are output when running in the "state-machine-cat.js" shell?

**Comment by sverweij** [comment]
Yes: https://github.com/sverweij/state-machine-cat/blob/main/tools/smcat-ast.schema.json
